fesvr/htif.h

```diff
@@ -93,6 +97,8 @@ class htif_t : public chunked_memif_t
        +signature=FILE\n\
       --chroot=PATH        Use PATH as location of syscall-servicing binaries\n\
        +chroot=PATH\n\
+      --payload=PATH       Load PATH memory as an additional ELF payload\n\
```

Comment:
Oops, I realised I missed out a word (*into* memory...). Do you want a trivial PR for that or will you fix it yourself?